Failed to construct terminal; falling back to unsupported

I get the following stack trace for most (all?) `mill` commands, but it seems to only show up the first time I run them. It doesn't seem to affect the actual command.

This is running in Arch using `alacritty` as the terminal.

```
[ERROR] Failed to construct terminal; falling back to unsupported
java.lang.NumberFormatException: For input string: "0x100"
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
        at java.lang.Integer.parseInt(Integer.java:580)
        at java.lang.Integer.valueOf(Integer.java:766)
        at jline.internal.InfoCmp.parseInfoCmp(InfoCmp.java:59)
        at jline.UnixTerminal.parseInfoCmp(UnixTerminal.java:242)
        at jline.UnixTerminal.<init>(UnixTerminal.java:65)
        at jline.UnixTerminal.<init>(UnixTerminal.java:50)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at java.lang.Class.newInstance(Class.java:442)
        at jline.TerminalFactory.getFlavor(TerminalFactory.java:211)
        at jline.TerminalFactory.create(TerminalFactory.java:102)
        at jline.TerminalFactory.get(TerminalFactory.java:186)
        at jline.TerminalFactory.get(TerminalFactory.java:192)
        at sbt.internal.util.ConsoleAppender$.ansiSupported(ConsoleAppender.scala:245)
        at sbt.internal.util.ConsoleAppender$.<init>(ConsoleAppender.scala:107)
        at sbt.internal.util.ConsoleAppender$.<clinit>(ConsoleAppender.scala)
        at sbt.internal.util.MainAppender$.defaultScreen(MainLogging.scala:61)
        at mill.scalaworker.ScalaWorker.compileScala(ScalaWorker.scala:168)
        at mill.scalalib.ScalaModule.$anonfun$compile$2(ScalaModule.scala:130)
        at mill.define.ApplyerGenerated.$anonfun$zipMap$9(ApplicativeGenerated.scala:21)
        at mill.define.Task$MappedDest.evaluate(Task.scala:358)
        at mill.eval.Evaluator.$anonfun$evaluateGroup$14(Evaluator.scala:297)
        at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
        at scala.Console$.withErr(Console.scala:192)
        at mill.eval.Evaluator.$anonfun$evaluateGroup$13(Evaluator.scala:297)
        at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
        at scala.Console$.withOut(Console.scala:163)
        at mill.eval.Evaluator.$anonfun$evaluateGroup$9(Evaluator.scala:296)
        at mill.eval.Evaluator.$anonfun$evaluateGroup$9$adapted(Evaluator.scala:254)
        at scala.collection.Iterator.foreach(Iterator.scala:929)
        at scala.collection.Iterator.foreach$(Iterator.scala:929)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1417)
        at scala.collection.IterableLike.foreach(IterableLike.scala:71)
        at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
        at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
        at mill.eval.Evaluator.evaluateGroup(Evaluator.scala:254)
        at mill.eval.Evaluator.evaluateGroupCached(Evaluator.scala:179)
        at mill.eval.Evaluator.$anonfun$evaluate$2(Evaluator.scala:86)
        at mill.eval.Evaluator.$anonfun$evaluate$2$adapted(Evaluator.scala:83)
        at scala.collection.Iterator.foreach(Iterator.scala:929)
        at scala.collection.Iterator.foreach$(Iterator.scala:929)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1417)
        at mill.eval.Evaluator.evaluate(Evaluator.scala:83)
        at mill.main.RunScript$.evaluate(RunScript.scala:216)
        at mill.main.RunScript$.$anonfun$evaluateTasks$1(RunScript.scala:202)
        at scala.util.Either.map(Either.scala:350)
        at mill.main.RunScript$.evaluateTasks(RunScript.scala:201)
        at mill.main.RunScript$.$anonfun$runScript$3(RunScript.scala:57)
        at ammonite.util.Res$Success.flatMap(Res.scala:61)
        at mill.main.RunScript$.runScript(RunScript.scala:56)
        at mill.main.MainRunner.$anonfun$runScript$1(MainRunner.scala:46)
        at ammonite.MainRunner.watchLoop(Main.scala:350)
        at mill.main.MainRunner.runScript(MainRunner.scala:34)
        at mill.Main$.main(Main.scala:66)
        at mill.Main.main(Main.scala)
```

--- a/build.sc
+++ b/build.sc
@@ -80,7 +80,8 @@ object core extends MillModule {
 
   def ivyDeps = Agg(
     ivy"com.lihaoyi::sourcecode:0.1.4",
-    ivy"com.lihaoyi:::ammonite:1.0.3-49-7fa03d0"
+    ivy"com.lihaoyi:::ammonite:1.0.3-49-7fa03d0",
+    ivy"jline:jline:2.14.5"
   )
 
   def generatedSources = T {

